modules/ContactsNitroModule/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "contacts-nitro-module",
+  "version": "0.0.1",
+  "main": "src/index",
+  "react-native": "src/index",
+  "description": "React Native Contacts Module with Nitro optimization",
+  "source": "src/index",
+  "files": [
+    "src",
+    "react-native.config.js",
+    "lib",
+    "android/build.gradle",
+    "android/gradle.properties",
+    "android/CMakeLists.txt",
+    "android/src",
+    "ios/**/*.h",
+    "ios/**/*.m",
+    "ios/**/*.mm",
+    "ios/**/*.cpp",
+    "ios/**/*.swift",
+    "app.plugin.js",
+    "*.podspec",
+    "README.md"
+  ],
+  "scripts": {
+    "postinstall": "tsc || exit 0;",
+    "typecheck": "tsc --noEmit",
```

Comment:
typecheck and lint seem like great examples of things that should inherit configs from the parent project. Still not at all convinced that adding a separate subproject is necessary or good in this case.